src/avio.jl

```diff
@@ -1,10 +1,10 @@
 # AVIO
 
-import Base: read, read!, show, close, eof, isopen, seekstart
+import Base: read, read!, show, close, eof, isopen, seek, seekstart
 
 export read, read!, pump, openvideo, opencamera, playvideo, viewcam, play
 
-using Compat
+using Compat, FileIO
```

Comment:
I'm curious what you're using from `FileIO`?  Is this the `imwrite` => `save` change in `test/avio.jl`?  If so, then   this import only needs to be in the test file, and not here.  Or was something broken before and fixed by this statement?